topo/node/juniper/juniper_test.go

```diff
@@ -156,6 +156,20 @@ func TestGenerateSelfSigned(t *testing.T) {
 			ni:       ni,
 			testFile: "testdata/generate_certificate_failure",
 		},
+		{
```

Comment:
please stub the sleep duration like line 133

Reply:
Done. I see now test does not wait.